generator/discovery/update_discovery_doc.sh

```diff
@@ -19,6 +19,26 @@ set -euo pipefail
 source "$(dirname "$0")/../../ci/lib/init.sh"
 source module ci/lib/io.sh
 
+function print_service_textproto() {
+  service_proto_path=$(echo "${1}" | sed -En 's/protos\/(google\/cloud\/compute\/.*\/v[[:digit:]]\/.*\.proto)/\1/p')
+  product_path=$(echo "${1}" | sed -En 's/protos\/(google\/cloud\/compute\/.*\/v[[:digit:]])\/.*\.proto/\1/p')
+  initial_copyright_year=$(date +"%Y")
+  echo "  rest_services {"
+  echo "    service_proto_path: \"${service_proto_path}\""
+  echo "    product_path: \"${product_path}\""
+  echo "    initial_copyright_year: \"${initial_copyright_year}\""
+  echo "    retryable_status_codes: [\"kUnavailable\"]"
+  echo "    generate_rest_transport: true"
+  echo "    generate_grpc_transport: false"
+  echo "  }"
```

Comment:
Consider a heredoc, e.g.:

```suggestion
  cat <<_EOF_
  rest_services {
    service_proto_path: "${service_proto_path}"
    product_path: "${product_path}"
    initial_copyright_year: "${initial_copyright_year}"
    retryable_status_codes: ["kUnavailable"]
    generate_rest_transport: true
    generate_grpc_transport: false
  }
_EOF_
```